Investigation/Java/web/automl/automl-web/src/main/resources/application-dev.yaml

```diff
@@ -0,0 +1,16 @@
+spring:
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/automl?characterEncoding=utf-8&useSSL=false&allowPublicKeyRetrieval=true
```

Comment:
We have a specific ip and port after we deploy to personal local Minikube

Reply:
> We have a specific ip and port after we deploy to personal local Minikube

@jia9iZhang Good point! Please resolve the comments in your PR when you have time so that Calm can do a integration test before this PR.